mne/decoding/transformer.py

```diff
@@ -230,8 +230,16 @@ def inverse_transform(self, epochs_data):
         This function makes a copy of the data before the operations and the
         memory usage may be large with big data.
         """
+        squeeze = False
+        # Can happen with CSP
+        if epochs_data.ndim == 2:
+            squeeze = True
+            epochs_data = epochs_data[..., np.newaxis]
         assert epochs_data.ndim == 3, epochs_data.shape
-        return _sklearn_reshape_apply(self._scaler.inverse_transform, True, epochs_data)
+        out = _sklearn_reshape_apply(self._scaler.inverse_transform, True, epochs_data)
+        if squeeze:
+            out = out[..., 0]
```

Comment:
safer to actually use `np.squeeze(..., axis=-1)` ?  it will error IFF the specified axis is not a singleton

Reply:
We should be guaranteed it's the right thing because of the corresponding logic where we `np.newaxis` in the same location above. I slightly prefer the current symmetry of the `[..., np.newaxis]` followed by `[..., 0]` view expansion and contraction